config.example/playbooks/example.yml

```diff
@@ -0,0 +1,14 @@
+---
+# Your custom playbooks should go in your DeepOps configuration directory,
+# under `config/playbooks`.
+# These playbooks can be used to make any customizations to your cluster
+# that aren't already provided by DeepOps!
+# For example, this playbook installs `cowsay` on all your Kubernetes nodes:
```

Comment:
Since this is for new users, should we link them to the Ansible docs?  I could go either way on this one. https://docs.ansible.com/ansible/latest/user_guide/playbooks.html

Reply:
Throwing the Ansible docs in here is a good idea, yes.